open-hackathon-client/src/client/static/js/manage.controller.js

```diff
@@ -1624,7 +1625,7 @@ angular.module('oh.controllers', [])
         pre_allocate_enabled: false,
         pre_allocate_number: 1,
         freedom_team: false,
-        login_provider: 63,
+        login_provider: 127,
```

Comment:
Gitcafe login is not valid now. And we don't want to fix it. Please remove the gitcafe login from the default